app/assets/images/icons/logo-desktop.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/

```

Comment:
As @jclopezdev mentions, do you need this file?

Reply:
Its a autogenerated file, if we delete it, it reapears

Reply:
But this shouldn't be marked as resolved just like that, what have you tried to solve it? These files must not be in the repository

Reply:
I've deleted the files, they are no longer in the repository

Reply:
Done